src/app/assets/languages/en.json

```diff
@@ -718,11 +718,17 @@
     "data-access-stats-tooltip": "Data access statistics for all caches across the cluster.",
     "data-access-subtitle": "Data access",
     "data-access-hits": "Retrieval Hits",
+    "data-access-hits-info": "Retrieval Hits-info",
```

Comment:
@dvagnero Need docs review here for tooltip.

Reply:
I added some info myself, can be reviewed later